Orange/widgets/unsupervised/owdistancetransformation.py

```diff
@@ -52,6 +52,8 @@ def __init__(self):
 
         gui.auto_commit(self.controlArea, self, "autocommit", "Apply",
                         checkbox_label="Apply on any change")
+        gui.separator(box)
+        box.layout().addWidget(self.report_button)
```

Comment:
I think this is not necessary, it puts the button above the Apply button, which is unlike the rest of the widgets, where Report is the last button in the control area.
![screenshot - 03142016 - 08 54 50 pm](https://cloud.githubusercontent.com/assets/684364/13757981/427024c6-ea27-11e5-8ffd-4321643720a7.png)

I know, you tried to do what @janezd said about aligning the buttons in some other PR, but this as-is is really unacceptable.